tools/provision.sh

```diff
@@ -18,9 +18,8 @@ elif [[ $ID == rhel || $ID == centos ]] && [[ ${VERSION_ID%.*} == 9 ]]; then
     curl -LO --insecure https://hdn.corp.redhat.com/rhel8-csb/RPMS/noarch/redhat-internal-cert-install-0.1-23.el7.csb.noarch.rpm
     sudo dnf install -y ./redhat-internal-cert-install-0.1-23.el7.csb.noarch.rpm dnf-plugins-core
     sudo dnf copr enable -y copr.devel.redhat.com/osbuild-team/epel-el9 "rhel-9.dev-$ARCH"
-    # koji is not available yet apparently
     # jmespath required for json_query
-    sudo dnf install -y ansible-core python3-jmespath
+    sudo dnf install -y ansible-core koji python3-jmespath
```

Comment:
shouldn't this be in koji.sh ? Why install it here if it is only needed for one test script only ?

Reply:
I suppose we do this to have the test dependencies in one place? AFAIK ansible is only used in ostree and installer tests... I'm ok with moving it to the koji.sh script if you think that makes more sense, but I think it doesn't really matter...

Reply:
Bleeeh. At this point it doesn't matter. The correct course of action will be to adjust all of the dependencies but this is outside of the current scope. Merging as-is.